lambdas/postcode-lookup/src/test/java/uk/gov/di/ipv/cri/address/api/handler/PostcodeLookupHanderTest.java

```diff
@@ -107,7 +108,8 @@ void AnyOtherExceptionThrows500()
             throws PostcodeLookupValidationException, PostcodeLookupProcessingException,
                     SessionExpiredException, SessionValidationException, SessionNotFoundException {
 
-        setupEventProbeErrorBehaviour();
+        when(eventProbe.counterMetric(anyString(), anyDouble())).thenReturn(eventProbe);
```

Comment:
This amend can be undone (see comment for `setupEventProbeErrorBehaviour()` below